roles/devscripts/templates/forwarding.yaml.j2

```diff
@@ -8,3 +8,6 @@ spec:
     ovnKubernetesConfig:
       gatewayConfig:
         ipForwarding: Global
+{% if host_routing %}
```

Comment:
host_routing|bool else it will get true with cifmw_devscripts_host_routing: "false"

Reply:
Good point. In the next patch, let me ensure the value set for `host_routing` is a boolean one.